src/supportive_defaults.jl

```diff
@@ -49,6 +55,45 @@ function supportive_defaults(cfg_symb::Symbol)
         return (; color = :deepskyblue3)
     elseif cfg_symb == :labels_grid_default
         return (; color = :gray, fontsize = 12, align = (:left, :top), space = :relative)
+        # plot_topoplot 
+    elseif cfg_symb == :topo_default_single
+        return (;
+            width = Relative(1),
+            height = Relative(1),
+            halign = 0.05,
+            valign = 0.95,
+            aspect = 1,
+        )
+    elseif cfg_symb == :topo_attributes_default
```

Comment:
- why not topo_default_attributes?
- is the default even necessary throughout here?
- can't we add these to `axis` in plot_config - at least for e.g. topoplot series?

Reply:
good quesion, I will play with configs later

Reply:
"It is not easy to run Matlab as a GitHub action. Therefore, we provided execution times of three consecutive executions."
is such text okay or something else needed?

Reply:
default is necessary (I guess) because we want to control interpolation
I think the main reason not to include it to config.axis is that by ?Axis you will not get documentation for it. You need ?Topoplot.topoplot

Reply:
also you cannot just do `Axis(f[1, 1]; config.axis...)` with interpolation, because it doesn't recognize such attributes

Reply:
so I would vote for keeping it